airflow/sensors/base.py

```diff
@@ -122,6 +135,7 @@ def __init__(
         self.timeout = timeout
         self.mode = mode
         self.exponential_backoff = exponential_backoff
+        self.max_wait = max_wait if max_wait is None else coerce_timedelta(max_wait, key="max_retry_delay")
```

Comment:
i would get rid of `key=...` it's confusing and not nec esp if you validate type in init

Reply:
I will try to move `coerce` code  to `_validate_input_values`

Reply:
while this method is used only for `max_wait`, I deleted the argument `key` and I renamed the method to `_coerce_max_wait`